+NeuroAnalysis/+Ripple/Prepare.m

```diff
@@ -134,31 +141,31 @@
                 dataset.lfp.data = Data;
                 dataset.lfp.fs = nsAnalogInfo.SampleRate;
                 dataset.lfp.electrodeid = ElectrodeID.LFP;
-                dataset.lfp.time = (ns2TimeStamps/nsAnalogInfo.SampleRate)*1000; % Convert to ms
+                dataset.lfp.time = (ns2TimeStamps/nsAnalogInfo.SampleRate);
```

Comment:
I guess plotdata plugin functions all use second as the time unit, which I think ms is more appropriate for neural signal.
Anyway, I could adapt easily when dataset is loaded.